WordPress/src/main/java/org/wordpress/android/ui/about/AboutWordPressActivity.kt

```diff
@@ -0,0 +1,34 @@
+package org.wordpress.android.ui.about
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.automattic.about.model.AboutConfig
+import com.automattic.about.model.AboutConfigProvider
+import com.automattic.about.model.HeaderConfig
+import com.automattic.about.model.LegalConfig
+import com.automattic.about.model.SocialsConfig
+import org.wordpress.android.R
+
+class AboutWordPressActivity : AppCompatActivity(), AboutConfigProvider {
+    @SuppressWarnings("RegexpSingleline")
+    override fun getAboutConfig(): AboutConfig {
+        return AboutConfig(
+                headerConfig = HeaderConfig.fromContext(this),
+                socialsConfig = SocialsConfig(
+                        instagramUsername = "wordpressdotcom",
+                        twitterUsername = "wordpressdotcom"
+                ),
+                legalConfig = LegalConfig(
+                        tosUrl = "https://wordpress.com/tos/",
+                        privacyPolicyUrl = "https://automattic.com/privacy/",
+                        acknowledgementsUrl = "file:///android_asset/licenses.html"
+                ),
+                onDismiss = { onBackPressed() }
+        )
+    }
```

Comment:
Notes for future PRs: 

- We should consider implementing this with a proper `ViewModel`.
- We should consider which values to provide when building the Jetpack app.